Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -686,10 +686,11 @@ struct HTTP2FrameDecoder {
 
         let priorityData: HTTP2Frame.StreamPriorityData?
         if flags.contains(.priority) {
-            let raw: UInt32 = bytes.readInteger()!
+            let values = bytes.readMultipleIntegers(as: (UInt32, UInt8).self)!
```

Comment:
This can be cleaned up as:

```suggestion
            let (raw, weight) = bytes.readMultipleIntegers(as: (UInt32, UInt8).self)!
```